src/content/guides/pay-payment-app-integration.mdx

```diff
@@ -0,0 +1,139 @@
+
+### Pay the Payment Request
+
+If you’ve determined that the Payment Request is payable with your Asset then you
+should display the Payment Request details to the user. Some relevant fields to
+display are:
+
+| Field        | Used for                                                                                 |
+| ------------ | ---------------------------------------------------------------------------------------- |
+| merchantName | The name of the merchant, user’s should always be made aware of who they are paying.     |
+| value        | The user must be aware of the amount and currency needed to fulfill the Payment Request. |
+
+Once a user has seen the relevant information and confirmed payment you should invoke the
+[Pay Payment Request API](https://docs.centrapay.com/api/payment-requests#pay-a-payment-request)
+from the backend on behalf of the user. Passing your Third-party asset type and
+the authorization property.
+
+If the payment was successful then the status of the payment request will be `paid`
+and you can show confirmation of payment to the user.
+
+## Payment Extentions
+
+Extensions to the payment protocol can be used to enhance the users experience
+by providing alternative payment flows.
+
+### Partial Pay
+
+If both your app and the merchants POS support [Partial Pay](/guides/partial-payment-extension/)
+then you can allow the user to pay only a portion of the payment request and
+then pay the remaining amount with an alternative payment method.
+
+The `partialAllowed` flag on the payment request will determine if the merchant
+allows partial payments. If they don’t then you should only allow paying the entire
+payment request.
+
+To pay a partial amount of the payment request pass the mode and amount to the pay
+payment request API.
+
+| Field  | Used for                                                                                           |
+| ------ | -------------------------------------------------------------------------------------------------- |
+| mode   | For partial payments the mode should be `partial`                                                    |
+| amount | For partial payments you must provide the amount of the Payment Request you want to partially pay. |
+
+The `remainingAmount` property on the payment request can be used to show how much
+the user still needs to pay with an alternative method.
+
+## Testing
+
+Centrapay supports test resources that can be used for integration testing. A test
+merchant will  be provided that accepts test payments. If you have supplied a
+test third-party asset then this merchant can be configured to accept it as a
+payment method.
+
+The [Centrapay Business Portal](https://app.centrapay.com/business) or APIs can
+be used to mock a real payment terminal to test your application.
```

Comment:
```suggestion
be used to test a real payment terminal to test your application.
```

Reply:
The sentence doesn't make sense with that change but I'll reword

Reply:
Sweet - just wanted to avoid technical terminology so that these guides are accessible to more people